tests/unit/FilingHistoryList.spec.ts

```diff
@@ -699,7 +699,26 @@ describe('Filing History List - redirections', () => {
     fileCorrectionItem.trigger('click')
     await flushPromises()
 
+    // verify to display the correction dialog modal
+    const fileCorrectionDialog = wrapper.find('#correction-filing-dialog')
+    expect(fileCorrectionDialog).toBeDefined()
+
+    // verify radio buttons for client and staff
+    const clientRadioBtn = wrapper.find('#correct-client-radio')
+    expect(clientRadioBtn).toBeDefined()
+
+    // click the staff radio button
+    const staffRadioBtn = wrapper.find('#correct-staff-radio')
+    expect(staffRadioBtn).toBeDefined()
+    staffRadioBtn.trigger('click')
+    await flushPromises()
```

Comment:
You should be able to await the trigger on the previous line and get rid of this flushPromises.

(But I know it doesn't always work.)

Reply:
Fixed. Thanks.

Reply:
I expected you to `await` the trigger on line 713.

Also please update lines 718/719 if it works.

Reply:
![image](https://user-images.githubusercontent.com/51458187/182957031-14b53576-08a8-4fdf-bfde-47269152a9aa.png)

Line 713 works with `await staffRadioBtn.trigger('click')`, but with a small warning. 

![image](https://user-images.githubusercontent.com/51458187/182957203-1ac314a4-b78b-4cdb-9dd8-6ce8fd333454.png)

Line 19 will not work with `await startCorrectionBtn.trigger('click')` if `await flushPromises()` is removed at line 718.

Reply:
OK, then go back to what you had. I think this is due to an older version of the test utils package. It works in other UIs.

Reply:
Working wxample from Edit UI:
![image](https://user-images.githubusercontent.com/10002889/182957671-9e8d5396-7f89-48cd-94bd-db54ed20c567.png)

Reply:
Filings UI:
`"@vue/test-utils": "1.0.0-beta.28",`

Edit UI:
`
"@vue/test-utils": "1.1.4",
`

But it's not as easy as just updating :(

Reply:
https://app.zenhub.com/workspaces/entities-team-space-6143567664fb320019b81f39/issues/bcgov/entity/13232

Reply:
Sure. Fixed.